pkg/imageengine/buildah/init.go

```diff
@@ -0,0 +1,96 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buildah
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/sealerio/sealer/pkg/auth"
+)
+
+const (
+	buildahEtcRegistriesConf = `
+[registries.search]
+registries = ['docker.io']
+
+# Registries that do not use TLS when pulling images or uses self-signed
+# certificates.
+[registries.insecure]
+registries = []
+
+[registries.block]
+registries = []
+`
+
+	builadhEtcPolicy = `
+{
+    "default": [
+	{
+	    "type": "insecureAcceptAnything"
+	}
+    ],
+    "transports":
+	{
+	    "docker-daemon":
+		{
+		    "": [{"type":"insecureAcceptAnything"}]
+		}
+	}
+}`
+
+	sealerAuth = `
+{
+	"auths": {}
+}
+`
+)
+
+// TODO do we have an util or unified local storage accessing pattern?
+func writeFileIfNotExist(path string, content []byte) error {
```

Comment:
Quite strange for the function definition. We usually create one if it does not exist, but we seldom write it it the path does not exist. Maybe we could separate the function into two parts, createFileIfNotExist and writefile.